news/2019-12-26-introducing-beatmap-pack-management.md

```diff
@@ -0,0 +1,71 @@
+
+### The Embodiment of Scarlet Devil pack
+
+*created by [celerih](https://osu.ppy.sh/users/4696296), [Uberzolik](https://osu.ppy.sh/users/1314547), [Venix](https://osu.ppy.sh/users/5999631), and [Xilver15](https://osu.ppy.sh/users/3099689)*  
+*written by [celerih](https://osu.ppy.sh/users/4696296)*  
+[Download](https://osu.ppy.sh/beatmaps/packs/1729)
+
+Available to download now, this pack is the first of the Touhou game packs, where a selection of various Touhou maps from a specific game is picked and curated just for you! This pack contains beatmaps from the first non-PC-98 Touhou game, and the sixth game in the series: The Embodiment of Scarlet Devil. The pack aims to offer a variety of styles and themes, ranging from 2008 to 2018 to offer something enjoyable for everyone. Packs for future games are currently in progress and should be released in the near future. Enjoy!
+
+### Best of 2010–2011 pack
+
+*created by [Dada](https://osu.ppy.sh/users/9119507), [\_p0ke\_](https://osu.ppy.sh/users/5434711), [Halfslashed](https://osu.ppy.sh/users/4598899), [Uberzolik](https://osu.ppy.sh/users/1314547), and [Cynplytholowazy](https://osu.ppy.sh/users/3901754)*  
+*written by [\_p0ke\_](https://osu.ppy.sh/users/5434711)*  
+[Download](https://osu.ppy.sh/beatmaps/packs/1733)
+
+Ever wanted to see what popular maps looked like in earlier eras of the game? Well now you can—you can check out some of the best maps that the game had to offer during 2010 and 2011 in a single pack! While there are a lot of great maps, we didn't have enough room to include everything, so hopefully you can find something on this list that you will enjoy. Play through them all and experience a blast from the past!
+
+### osu!dan project
+
+*written by [celerih](https://osu.ppy.sh/users/4696296)*
+
+The osu!dan project is a brand new community created project that seeks to provide a "campaign mode" to the game. Players will progress through levels of increasing difficulty in a skill of their choice, such as aim, stamina, accuracy, et cetera.
+
+Each of the dans will contain 20 levels of increasing difficulty. In order to complete a level, the player must reach the requirements on the maps presented, such as a certain accuracy threshold or a full combo. For now, the release of dan levels is purely for fun or practice, but a more official implementation is planned for the future. ;)
+
+The first of the dans is available to download now on the Beatmap Packs listing. The skill tested in this pack is aim and will surely push your circle clicking to the limits. The requirements for this dan are quite simple: you need to achieve full combo on 10 out of 11 maps in the dan to progress to the next level. Once again, this for now is just for your own practice, or to compare yourself with your friends—there is no implementation of tracking your progress, for now.
```

Comment:
more packs available now
also none of them seem to be listed on the packs page so you should link them individually under celerihs name

Reply:
They'll be released today